### PR TITLE
Tokenize ShimmerView

### PR DESCRIPTION
**Pull request by jeaniehuynh**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

Tokenization of ShimmerView. Note that this change changes APIs and is thus a breaking change.

- ios/FluentUI.Demo/FluentUI.Demo/Demos.swift
  - Move ShimmerView demo controller to vNext section
- ios/FluentUI.Demo/FluentUI.Demo/Demos/ShimmerLinesViewDemoController.swift
  - Add new parameters to ShimmerView and ShimmerLinesView inits
- ios/FluentUI.xcodeproj/project.pbxproj
  - Add ShimmerTokens file
- ios/FluentUI/Shimmer/ShimmerLinesView.swift
  - Add lineCount, firstLineFillPercent, and lastLineFillPercent to ShimmerLines init
  - Update to use tokens where available
- ios/FluentUI/Shimmer/ShimmerTokens.swift
  - Add tokens for Shimmer and ShimmerLines
- ios/FluentUI/Shimmer/ShimmerView.swift
  - Move ShimmerStyle to ShimmerTokens
  - Add shimmerStyle, shimmersLeafViews, and usesTextHeightForLabels to Shimmer init
  - Update to use tokens where available

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| https://user-images.githubusercontent.com/31874971/173157760-a5348f0f-c079-4e50-9758-c4b87f79100f.mp4 | https://user-images.githubusercontent.com/31874971/173157570-7ac11d4e-d972-48ab-b7df-1b2de91ee3be.mp4 |
### Pull request checklist

This PR has considered:
- [x] Light and Dark appearances
- [x] iOS supported versions (all major versions greater than or equal current target deployment version)
- [x] VoiceOver and Keyboard Accessibility
- [x] Internationalization and Right to Left layouts
- [x] Different resolutions (1x, 2x, 3x)
- [x] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1021)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -5,14 +5,20 @@ on:
     branches:
       - main
       - main_0.1
-      - vnext-prototype
-      - vnext-tokens
+      - main_0.2
```
Comment (edjamesmsft) [inline .github/workflows/ci.yml:8]: Should this CP be done through fluent2-tokens, THEN merged into this PR?

Comment (HyperSkys) [reply .github/workflows/ci.yml:8]: ok

**Comment by Saadnajmi** [comment]
@harrieshin should we have an item for FURN to match tokens?

**Comment by HyperSkys** [comment]
> @harrieshin should we have an item for FURN to match tokens for an eventual "v1" shimmer ?

what the hell is a furn

**Pull request closed without merging**